src/components/KMultiselect/KMultiselect.vue

```diff
@@ -1015,12 +1016,17 @@ watch(() => props.items, (newValue, oldValue) => {
 }, { deep: true, immediate: true })
 
 const numericWidth = ref<number>(300)
-const setNumericWidth = (): void => {
+const setNumericWidth = async (): Promise<void> => {
+  numericWidth.value = 300
```

Comment:
Is this intentional as the default value is already initialized to 300?

Reply:
Yes it is - it's needed to reset to default value before we assign the value of element width on line 1022